apps/iis.go

```diff
@@ -26,13 +26,32 @@ type MetricsReceiverIis struct {
 	confgenerator.ConfigComponent `yaml:",inline"`
 
 	confgenerator.MetricsReceiverShared `yaml:",inline"`
+	Version                             string `yaml:"receiver_version"`
```

Comment:
[Optional] If #554 gets merged, we can pull `Version` out of the `mssql` and `iis` receivers into a new type e.g. `MetricsReceiverVersionShared` and reuse it in the `mssql` and `iis` receivers, just so we don't have to redefine the `Version` field each time.